### PR TITLE
Skinning

### PR DESCRIPTION
**Pull request by webermm**
Added documentation for skinning feature in v1.6.0.

### CODE REVIEW

File: doc/advanced/skinning/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+When importing meshes using multiple materials several MeshNodes will be created. The Skin object will then contain a list of all these MeshNodes as targets.
+
+To control the meshnode deformation via the shader the target meshnode set in the Skin object needs to have a private material with the correct uniform for the transformation matrices as described in detail below. The Skin object will therefore still show an error directly after the import:
+
+![](./docs/skin-property-browser.png)
+
+
+## Setting up the shader
+
+As a starting point for setting up a shader for a skinned mesh a template shader is included in the RamsesComposer distribution in the `resources/shaders/` subfolder. The fragment shader is
```
Comment (violinyanev) [inline doc/advanced/skinning/README.md:55]:
```suggestion
As a starting point for setting up a shader for a skinned mesh a template shader is included in the RamsesComposer distribution in the `resources/shaders/` subfolder. The fragment shader can look like this for example:
```

File: doc/advanced/skinning/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+![](./docs/preview-1.png)
+
+
+It is now possible to deform the meshnode via changes in the position or orientation of the skeleton node hierarchy.
+
+
+## Control via animations
+
+However the exmple glTF file used already contains an animation which controls the rotation of the `node_2` in the skeleton node hierarchy via a link:
```
Comment (violinyanev) [inline doc/advanced/skinning/README.md:111]:
```suggestion
The exmple glTF file used already contains an animation which controls the rotation of the `node_2` in the skeleton node hierarchy via a link:
```

**Review by violinyanev — CHANGES_REQUESTED** [review]

**Review by violinyanev — DISMISSED** [review]

**Comment by violinyanev** [comment]
Can't have links in the PR, they don't work of course. See github actions result

File: doc/advanced/skinning/skinning-simple.rca [hunk 3]
```diff
@@ -0,0 +1,2206 @@
...
+                    }
+                },
+                "stdModules": {
+                    "base": true,
+                    "debug": true,
+                    "math": true,
+                    "string": true,
+                    "table": true
+                },
+                "uri": "../../../../../OneDrive - Paradox Cat GmbH/Documents/RamsesComposer/skinning/scripts/clip-timer.lua"
```
Comment (ghost) [inline doc/advanced/skinning/skinning-simple.rca]: This script seems to be missing.

**Comment by ghost** [comment]
PR closed since changes are merged in main Ramses Composer development repo.

**Pull request closed without merging**